src/core/remote.js

```diff
@@ -539,8 +539,18 @@ Remote.prototype.getLedgerSequence = function(callback = function() {}) {
     // the "current" ledger is the one after the most recently closed ledger
     callback(null, this._ledger_current_index - 1);
   } else {
+    let timedOut = false;
+    const timeout = setTimeout(() => {
+      timedOut = true;
+      callback(new RippleError('tejTimeout',
```

Comment:
I think I've said before that tej* errors are not general-purpose; they are specifically transaction results generated from ripple-lib